eng/Versions.props

```diff
@@ -276,7 +276,7 @@
     <XunitExtensibilityCoreVersion>$(XunitVersion)</XunitExtensibilityCoreVersion>
     <XunitExtensibilityExecutionVersion>$(XunitVersion)</XunitExtensibilityExecutionVersion>
     <XUnitRunnerVisualStudioVersion>2.4.3</XUnitRunnerVisualStudioVersion>
-    <MicrosoftDataSqlClientVersion>1.0.19249.1</MicrosoftDataSqlClientVersion>
+    <MicrosoftDataSqlClientVersion>4.0.1</MicrosoftDataSqlClientVersion>
```

Comment:
That's a big jump. Anything to keep an 👁️ out for @smitpatel @bricelam❔

Reply:
@smitpatel @bricelam any concerns here? @BrennanConroy any other testing I should do?

Reply:
🤷 Just make sure to make an announcement about potential breaking changes with the version update.

Reply:
https://github.com/aspnet/Announcements/issues/481